core/command.js

```diff
@@ -116,6 +116,49 @@ CKEDITOR.command = function( editor, commandDefinition ) {
 		return allowed = editor.activeFilter.checkFeature( this );
 	};
 
+	/**
+	 * Provides command's label string. Label is define with few fallbacks in specific order described below:
+	 *
+	 * 1. If {@link CKEDITOR.commandDefinition#getLabel} method was defined, it is used to provide label,
+	 * 2. If {@link CKEDITOR.commandDefinition#label} atribute exists, it is used to provide label.
+	 * 3. If {@link CKEDITOR.command#uiItems} is non empty, then label of first array's item is used as label
+	 * 4. Empty string.
+	 *
+	 * @since 4.8.0
+	 * @returns {String} Command's label
+	 */
+	this.getLabel = function() {
+		if ( typeof commandDefinition.getLabel == 'function' ) {
+			return commandDefinition.getLabel.call( this, editor );
+		} else if ( commandDefinition.label ) {
+			return commandDefinition.label;
+		} else if ( this.uiItems && this.uiItems.length && this.uiItems[ 0 ].label ) {
+			return this.uiItems[ 0 ].label;
```

Comment:
This kind of fallback allows on reusing already created labels for buttons without modifying all other plugins.
So steps are to use `getLabel` method if exist, then `label` property, and then button's label if exist. So we use translations for button's label such as **bold**, *italic*, etc, without waiting for them being translate and just use what we have already.